Automattic/woocommerce-payments
# Update woopay logo

Updates the logo used for WooPay on the express checkout button as well as the settings page.

![woopay-logo-compare](https://user-images.githubusercontent.com/1558827/228646155-1aa23b47-b9ca-4f5e-978f-a00706d49b9c.png)

#### Testing instructions

* Make sure the logo is updated on the express checkout button as well as the various settings pages.

-------------------


**Post merge**
